puppet/precip/manifests/php.pp

```diff
@@ -11,6 +11,7 @@
       gd         => { },
       imagick    => { },
       intl       => { },
+      mbstring   => { },
```

Comment:
Yeah, idk why we never had mbstring set up. It wasn't even a matter of missing it when everything about PHP was refactored late last year, it was just never there. Whatever, it's there now.